app/assets/javascripts/directives/project_roles_directive.js

```diff
@@ -0,0 +1,13 @@
+samson.directive('projectRoles', function(userProjectRoleFactory) {
+  return {
+    restrict: 'E',
+    templateUrl: 'shared/_project_roles.tmpl.html',
+    scope: {
+      roles: '=',
+      roleChanged: '&'
+    },
+    link: function($scope, element) {
+      $scope.project_role = userProjectRoleFactory.build(element[0]);
```

Comment:
I know, it really looks like a code smell. Tried to find a way to avoid it but with no success. The current element should be just that, the element, not an array with the root element of the directive. :(